pom.xml

```diff
@@ -91,10 +91,9 @@
         <mockito.version>4.3.1</mockito.version>
         <brapi-java-client.version>2.1-SNAPSHOT</brapi-java-client.version>
         <commons-io.version>2.11.0</commons-io.version>
```

Comment:
Is the commons-io.version needed?

Reply:
It was unnecessary, I removed it. Thanks.